docker/ci/Dockerfile

```diff
@@ -8,7 +8,7 @@
 
 
 #########################################################
-FROM ubuntu:20.04 as base
+FROM ubuntu:20.04 AS base
```

Comment:
I was getting warnings on build:

```
 3 warnings found (use --debug to expand):
 - FromAsCasing: 'as' and 'FROM' keywords' casing do not match (line 11)
 - FromAsCasing: 'as' and 'FROM' keywords' casing do not match (line 53)
 - FromAsCasing: 'as' and 'FROM' keywords' casing do not match (line 70)
```

Seemed easy enough to fix while I was in the area